apps/www/pages/launch-week/index.tsx

```diff
@@ -92,15 +92,6 @@ export default function TicketHome({ users, meetups }: Props) {
     }
   }, [supabase])
 
-  useEffect(() => {
-    toggleTheme(true)
-    document.body.className = 'dark bg-[#020405]'
```

Comment:
@saltcod the background in the launch week page should be `#020405`
Otherwise it looks odd with the default bg color.

<img width="1985" alt="Screenshot 2023-09-25 at 14 15 03" src="https://github.com/supabase/supabase/assets/25671831/e4bd36de-dc67-4e6f-b10d-ee3956fa1475">